src/test/rustdoc-ui/public-reexported-item-doc-test.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+
+#![deny(rustdoc::private_doc_tests)]
+
+mod foo {
+    /// re-exported doc test
+    ///
+    /// ```
+    /// assert!(true);
+    /// ```
+    pub fn bar() {}
+}
+
+pub use foo::bar;
```

Comment:
Can you add:

```rust
pub fn foo() {}

pub mod whatever {
    pub fn bar() {}
}
```

to be sure it doesn't warn on public items too please?

Reply:
Sure can :)